playbooks/common/openshift-cluster/upgrades/pre/verify_health_checks.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Verify Host Requirements
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: "upgrade"
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
```

Comment:
Will push updated `disk_availability` check that takes the `upgrade` context into account; halving the required disk space in that case